scripts/globalSetup.js

```diff
@@ -0,0 +1,3 @@
+const { version } = require('webpack');
+
+module.exports = () => console.log(`Running tests for webpack @${version}`);
```

Comment:
Can we improve its visibility? 

![Screenshot at 2020-11-03 13-28-39](https://user-images.githubusercontent.com/46647141/97961088-a139e700-1dd8-11eb-8b73-62b261cf8694.png)

```suggestion
module.exports = () => console.log(`Running tests for webpack @${version}`);
```
```suggestion
module.exports = () => console.log(`\n Running tests for webpack @${version} \n`);
```